cli/data.py

```diff
@@ -25,11 +25,15 @@
 from libs.datasets import dataset_utils
 from libs.datasets import combined_datasets
 from libs.datasets.sources import forecast_hub
+from libs.datasets import tail_filter
 from libs.us_state_abbrev import ABBREV_US_UNKNOWN_COUNTY_FIPS
 from pyseir import DATA_DIR
 import pyseir.icu.utils
 from pyseir.icu import infer_icu
 
+TailFilter = tail_filter.TailFilter
```

Comment:
what's the difference between this doing it with `from libs.datasets.tail_filter import TailFilter` ?

Reply:
Mostly following guidance from the [gsg](https://google.github.io/styleguide/pyguide.html#233-decision)